projects/plugins/jetpack/_inc/client/components/data/query-backup-undo-event/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import {
+	fetchBackupUndoEvent,
+	hasLoadedBackupUndoEvent,
+	isFetchingBackupUndoEvent,
+} from 'state/at-a-glance';
+
+class QueryBackupUndoEvent extends Component {
+	componentDidMount() {
+		if ( ! this.props.fetchingBackupUndoEvent && ! this.props.hasLoadedBackupUndoEvent ) {
+			this.props.fetchBackupUndoEvent();
+		}
+	}
+
+	render() {
+		return null;
+	}
+}
+
+QueryBackupUndoEvent.defaultProps = {
+	fetchBackupUndoEvent: () => {},
+};
+
+export default connect(
+	state => {
+		return {
+			fetchBackupUndoEvent: fetchBackupUndoEvent(),
```

Comment:
```suggestion
			fetchBackupUndoEvent,
```
Not sure/tested if it can be simplified or is good idea :)

Reply:
Makes sense! Just committed it :D